src/datasets/search/client_ltv/intro.md

```diff
@@ -41,6 +41,6 @@ The `normalized_ltv_ad_clicks_current` field, for example, does **not** represen
 
 LTV is broken down by engine, so the LTV for a user who searches on multiple engines must be interpreted in context. **LTV is only available for Google and Bing on Firefox Desktop** at this time.
 
-We **do** have the ability to calculate a dollar value per user, however the (unnormalized) table is restricted to those with proper revenue access. For more information, please reach out to :bmiroglio (bmiroglio@mozilla.com).
+We **do** have the ability to calculate a dollar value per user, however the (unnormalized) table is restricted to those with proper revenue access. For more information, please reach out to :frank (frank@mozilla.com).
```

Comment:
FWIW in general the guideline is not to mention specific individuals or teams unless absolutely necessary (basically for this reason) see https://docs.telemetry.mozilla.org/contributing/style_guide.html#writing-style

I'd personally suggest just telling people to ask on one of our default channels under getting help: https://docs.telemetry.mozilla.org/concepts/getting_help.html